$NetBSD: patch-ae,v 1.1 2006/06/26 11:14:56 rillig Exp $

Some dependency package also provides <audio/audio.h>, so it must be
included directly.

--- src/game/gui.c.orig	2003-12-13 18:01:24.000000000 +0100
+++ src/game/gui.c	2006-06-26 13:11:51.000000000 +0200
@@ -3,6 +3,7 @@
 #include <math.h>
 
 #include "game/gltron.h"
+#include "../include/audio/audio.h"
 
 /* FIXME: "ignored playMenuFX" */
 void playMenuFX(int foo) { }
